### ISSUE TITLE
Talend 7.4.1 compatibility

### ISSUE THREAD

**Issue by Seb2nim** [jlolling/talendcomp_tGoogleSheet#9]
tGoogleSheetOutput doenst work on Talend 7.4.1.
Unfortunatelly, 7.4.1 is the only version where MacOSX Big Sur (N-1) is supported.
Any chance for an update ?

`Exception in thread "main" java.lang.NoSuchMethodError: com.google.api.client.http.HttpTransport.isMtls()Z
	at com.google.api.services.drive.Drive$Builder.chooseEndpoint(Drive.java:10139)
	at com.google.api.services.drive.Drive$Builder.<init>(Drive.java:10172)
	at org.talend.components.google.drive.runtime.client.GoogleDriveService.getDriveService(GoogleDriveService.java:86)
	at org.talend.components.google.drive.runtime.GoogleDriveRuntime.getDriveService(GoogleDriveRuntime.java:179)
	at org.talend.components.google.drive.runtime.GoogleDriveListReader.<init>(GoogleDriveListReader.java:28)
	at org.talend.components.google.drive.runtime.GoogleDriveSource.createReader(GoogleDriveSource.java:72)
	at org.talend.components.google.drive.runtime.GoogleDriveSource.createReader(GoogleDriveSource.java:31)
	at kalorio.gd_01_pushtickets_0_1.GD_01_PushTickets.tGoogleDriveList_1Process(GD_01_PushTickets.java:1646)
	at kalorio.gd_01_pushtickets_0_1.GD_01_PushTickets.tGoogleDriveConnection_1Process(GD_01_PushTickets.java:849)
	at kalorio.gd_01_pushtickets_0_1.GD_01_PushTickets.tJava_1Process(GD_01_PushTickets.java:613)
	at kalorio.gd_01_pushtickets_0_1.GD_01_PushTickets.runJobInTOS(GD_01_PushTickets.java:5860)
	at kalorio.gd_01_pushtickets_0_1.GD_01_PushTickets.main(GD_01_PushTickets.java:5644)`

**Comment by jlolling** [comment]
Because Talend 7.4 will not continued I have not tested it with this version. I can tell you the current release of the component 4.1 released at 2021-01-27 works well in Talend 7.3.1 and Talend 8.0.1
But nevertheless I am surprised about this issue. 
Which java version do you use? I have also Big Sure running with Java 11.
Do you use this component along with other Google service related components?

**Comment by Seb2nim** [comment]
Hello Jan, Sincere apologies for late answer.
I have a running TOS DI 7.3.1 on my old iMac, so i had to make the job done.
Back on my MacBook to try to make it work.

I let 7.4.1 as it will not be continued. Fail to made it work on Big Sur.

I have a job with mixins tGoogleSheetOutput and original GoogleDrive Components (connexion, tGoogleDriveList, etc).

> iMac MacOsx High Sierea (10.13.6) + Talend DI 7.3.1 => OK

> Mac OSX Big Sur (MacBook Intel 16", jdk 11) + Talend 8.0.1 => Fail (exception in first comment).

I noticed precence of 2 versions of google-http-client-1.31.0 and 1.38.0 (+ jackson etc..) in project "temp" directory.
It seems tGSConnetion, tGSList etc uses another google-http-client version....

I tried using tGoogleSheetOutput alone in a job (with a previously retreived file_id) and it works...
